vars.yml.fips.example

```diff
@@ -0,0 +1,17 @@
+auditplan: ["operatorinstall"]
```

Comment:
Is a separate example necessary?

Reply:
Maybe not. I'll try to unify those and let the options commented.

Reply:
Done!